app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx

```diff
@@ -213,232 +161,87 @@ const SendBitcoinDestinationScreen = ({
     fetchPolicy: "cache-first",
     returnPartialData: true,
   })
-  const myUsername = data?.me?.username
+  const wallets = data?.me?.defaultAccount.wallets
   const bitcoinNetwork = data?.globals?.network
   const contacts = useMemo(() => data?.me?.contacts ?? [], [data?.me?.contacts])
 
   const { LL } = useI18nContext()
-  const [userDefaultWalletIdQuery] = useUserDefaultWalletIdLazyQuery()
-
-  const checkUsername:
-    | ((
-        username: string,
-      ) => Promise<{ usernameStatus: UsernameStatus; walletId?: string }>)
-    | null = useMemo(() => {
-    if (!contacts || myUsername === undefined) {
-      return null
-    }
-    const lowercaseContacts = contacts.map((contact) => contact.username.toLowerCase())
-    const lowercaseMyUsername = myUsername ? myUsername.toLowerCase() : ""
-    const getWalletIdForUsername = async (username: string) => {
-      const { data } = await userDefaultWalletIdQuery({ variables: { username } })
-      return data?.userDefaultWalletId
-    }
-    return async (username: string) =>
-      getUsernameStatus({
-        username,
-        contacts: lowercaseContacts,
-        myUsername: lowercaseMyUsername,
-        getWalletIdForUsername,
-      })
-  }, [contacts, userDefaultWalletIdQuery, myUsername])
+  const [userDefaultWalletIdQuery] = useUserDefaultWalletIdLazyQuery({
+    fetchPolicy: "cache-and-network",
```

Comment:
it's a lazy query. I don't think we want to use `cache and network`:
- because its a lazy query, it acts like a mutation and will only be called when the function is called instead of when the screen is loaded. 

so I'm not sure how we would get the network query backed from this function if the value is cached. I'll assume we don't

also, we may not want to cache this at all, so maybe `no-cache` is the appropriate solution.

Reply:
Yeah my one thinking for cache and network was that the call would always succeed instantly if it was in the cache, and then the next time it was called it would be the up to date value. The trade off would be that if someone changed default wallets, there would be one payment to them using the old wallet id. I suppose however that if the getWaleltId query fails there will probably be other things failing and it wont be worth it to try and prevent that situation.

Reply:
>  I suppose however that if the getWaleltId query fails there will probably be other things failing and it wont be worth it to try and prevent that situation.

exactly. 

I think this is like the getFee where we try to preemptively calculate them locally if this is from our node instead of calling the server. the benefits do not outweigh the added complexity